.github/workflows/patchworks.yaml

```diff
@@ -16,14 +23,25 @@ jobs:
     steps:
       - uses: actions/checkout@v3
 
-      - name: Get list of new patches
-        run: |
+      - name: Get timestamps and make directories
           TZ=UTC date -d "@$((($(date +%s) + 450) / 900 * 900))" '+%Y-%m-%dT%H:%M:%S' > date_recent.txt
           TZ=UTC date -d "@$((($(date +%s) - 450) / 900 * 900))" '+%Y-%m-%dT%H:%M:%S' > date_old.txt
           TZ=UTC date -d "@$((($(date +%s) - 1350) / 900 * 900))" '+%Y-%m-%dT%H:%M:%S' > date_older.txt
-
           mkdir patch_urls
           mkdir patchworks_urls
+
+      - name: Update times for workflow dispatch
+        if: ${{ inputs.backup_time != '' }}
+        run: |
+          TZ=UTC date -d "@$((($(date -d ${{ inputs.timestamp }} +%s) + 450) / 900 * 900))" '+%Y-%m-%dT%H:%M:%S' > date_recent.txt
```

Comment:
Do you need to wrap this in quotes? When I run locally I need quotes:
```
date -d "Thu, 28 Sep 2023 13:28:09" +%s
```

Reply:
updated to have quotes